Formula/luyten.rb

```diff
@@ -28,10 +28,14 @@ def install
   end
 
   test do
-    io = IO.popen("luyten")
-    sleep 5
-    Process.kill("SIGINT", io.pid)
-    Process.wait(io.pid)
-    io.read !~ /Exception/
+    begin
+      io = IO.popen("#{bin}/luyten 2>&1")
+      sleep 5
+    ensure
+      Process.kill("SIGINT", io.pid)
+      Process.wait(io.pid)
+    end
+    output = io.read
+    assert_no_match /Exception/, output if output !~ /HeadlessException/
```

Comment:
> Is there a way to check the exit code here?

Yes. `Process.wait` sets `$?` to the status of the process that is has been waiting for, thus you should be able to check with `$?.success?` (haven't tested it).